data/metadata.json

```diff
@@ -484,6 +484,7 @@
     "hack-toolbox/lockscreen/active": {
         "sound-file": "toolbox/lockscreenActive.wav",
         "loop": true,
+        "priority": true,
```

Comment:
I feel this should be called `prioritary` or use a number based priority.

Reply:
I think `prioritary` is harder to pronounce, and it seems `priority` is what is commonly used in English. Something can be a priority or not, so not sure using a Boolean is inherently bad here.

Note that I am not implementing levels in this version since is much more complex.

Reply:
@robinmt mentioned to use whether "priority" or "important"... so whichever of both are fine to me.